src/ralph/licences/models.py

```diff
@@ -36,8 +37,11 @@ class LicenceType(NamedMixin, models.Model):
     def create_from_string(cls, string_name, *args, **kwargs):
         return cls(name=string_name)
 
+    class Permissions:
+        blacklist = set()
```

Comment:
do you add empty `Meta` or 
```
class Meta:
    unique_together = []
```
to every model? Empty `blacklist` could be default option - only when you want to blacklist sth you could add `Permissions` class to model definition.